### PR TITLE
:bookmark: Commit version v2.1.0

### PR DESCRIPTION
**Pull request by patrikspiess**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ For unreleased changes see [WHATSNEW.md](WHATSNEW.md)
...
+
+- Option `--raw` for `fmg get devices`
+- Support for HTTP `PATCH` and `DELETE` method in FortiClientEMS
+- Option `--smtp` for `fgt config check`
+
+
+### Changed
+
+- `fmg get devices` also shows ha nodes if device is a cluster
+- Make `Fortinet.api()` more generic to support more methods
```
Comment (lucmurer) [inline CHANGELOG.md:25]: Perhaps be more specific here: `HTTP methods`

**Review by lucmurer — CHANGES_REQUESTED** [review]
1 change for perfectness ;-)

File: CHANGELOG.md (hunk 1 above)
Comment (patrikspiess) [reply CHANGELOG.md:25]: The complete sentence is:

- Support for HTTP `PATCH` and `DELETE` **method** in FortiClientEMS

So, the word "method" is already present after the methods.

Comment (lucmurer) [reply CHANGELOG.md:25]: Hmm I would nonetheless add the `HTTP` word in line 25, because this is to enable what then has been added.

Comment (patrikspiess) [reply CHANGELOG.md]: Yes, of course you're right. I looked at the wrong line.
Has been added now.

**Review by lucmurer — APPROVED** [review]

**Pull request merged**